metadatastore/commands.py

```diff
@@ -126,8 +126,8 @@ def insert_run_start(time, scan_id, beamline_id, beamline_config, uid=None,
     beamline_id : str
         Beamline String identifier. Not unique, just an indicator of
         beamline code for multiple beamline systems
-    beamline_config : metadatastore.odm_temples.BeamlineConfig
-        Foreign key to beamline config corresponding to a given run
+    beamline_config : str
+        uid of beamline config corresponding to a given run
```

Comment:
My only issue is that the kwargs to `insert_*` and `find_*` are not symmetric. On the `find_*` side, `run_start` refers to the whole mds document, while `run_start_uid` is the uid of the run_start. On the insert_* side, `run_start` is the uid of the run_start.  I guess it just feels a little strange, but other than that I'm not sure I can come up with a compelling argument